packages/api/core/src/api/make.ts

```diff
@@ -264,7 +264,7 @@ export const listrMake = (
                      *   * Change the entire API of maker from a single constructor to
                      *     providing a MakerFactory
                      */
-                    maker.prepareConfig(targetArch);
+                    await maker.prepareConfig(targetArch);
```

Comment:
I think we need
```suggestion
                    await Promise.resolve(maker.prepareConfig(targetArch));
```

in case a newer version of core acts on an older version of maker-base (shouldn't happen but we can pre-empt the issue here)